Scripts/Python/clftImager.py

```diff
@@ -441,7 +441,7 @@ def OnNotify(self,state,id,events):
                 PlayFull = 1
                 PlayFinal = 0
                 PtDebugPrint("play full opening speech")
-            elif (IsCleftSolved() and self.TPOTSolved()):
+            elif (StartInRelto() and self.TPOTSolved()):
```

Comment:
So, if I am reading this correctly, players who start on the cleft path can never see the TPotS Yeesha speech?

Reply:
`StartInRelto()` is true once they solve the Cleft (and then start in Relto for subsequent logins)

Reply:
StartInRelto() is false if you have solved the cleft.
This check is here because TPOTSolved() will error due the code not being generated until you make it to relto.